### PR TITLE
Example receiver for webhooks

### PR DESCRIPTION
**Pull request by mrexodia**
Follow up for #3517

### CODE REVIEW

**Comment by lunny** [comment]
@mrexodia Thanks for you pull request. As an example, I think maybe you can remove log file and config file. That will let it simple and clear enough for beginners.

**Comment by mrexodia** [comment]
Okay, will do!

**Review by techknowlogick — APPROVED** [review]

File: docs/content/doc/features/webhooks.en-us.md [hunk 1]
```diff
@@ -104,3 +104,203 @@ X-Gitea-Event: push
...
+		}
+	}()
+
+	//get the hook event from the headers
+	event := r.Header.Get("X-Gogs-Event")
+	if len(event) == 0 {
+		event = r.Header.Get("X-Gitea-Event")
+	}
+
+	//only push events are current supported
```
Comment (adelowo) [inline docs/content/doc/features/webhooks.en-us.md]: `s/current/currently`

Comment (mrexodia) [reply docs/content/doc/features/webhooks.en-us.md]: Should be fixed now

**Comment by mrexodia** [comment]
@lunny unfortunately I don't currently have the time to scrub the config file and other things you requested. Consider merging it anyway, since a production-tested example can still be useful in my opinion...

**Pull request closed without merging**